test/services/katello/managed_content_medium_provider.rb

```diff
@@ -0,0 +1,25 @@
+require 'katello_test_helper'
+
+module Katello
+  module Service
+    class ManagedContentMediumProviderTestBase < ActiveSupport::TestCase
+    end
+
+    class ManagedContentMediumProvider < ManagedContentMediumProviderTestBase
+      setup do
+        @org = FactoryBot.create(:katello_organization)
+        @distro = katello_repositories(:fedora_17_x86_64)
+      end
+
+      def test_unique_id
+        host = ::Host::Managed.new(:name => 'foobar', :managed => false, :organization => @org)
+        host.content_facet.kickstart_repository = @distro
+        host_group = ::Hostgroup.new(:name => 'bar')
+        host_group.kickstart_repository = @distro
+        ::Katello::ManagedContentMediumProvider.new(host).unique_id
```

Comment:
For the sake of readability, can you change it to assertion (for all three checks):
``` ruby
assert_not_nil ::Katello::ManagedContentMediumProvider.new(host).unique_id
```